### PR TITLE
First revision of ATSSS v2 section

### PR DESCRIPTION
**Pull request by markusa**

### CODE REVIEW

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -306,19 +312,41 @@ This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no
...
+However, UDP compared to TCP has different dimensions to be considered in
+respect to the ATSSS high-layer approach.
+
+1. UDP consists of a mix of traffic with different demands on transport
+2. UDP is stateless and non-reliable
+
+That does not allow a simple defintion of a multipath transport protocol for UDP
+like it was done for TCP purposes. Even if one would limit to support the QUIC,
+as dominating part of the UDP share, only, this raises issues by its nature of
+an end-to-end encrypted protocol. A potentially upcoming Multipath QUIC
```
Comment (boucadair) [inline draft-quic-atsss-reqs.mkd]: How is that different from handling TLS connections?

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -306,19 +312,41 @@ This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no
...
+{{I-D.deconinck-quic-multipath}} cannot be used in the ATSSS architecture like
+the MPTCP in {{fig-mptcp-arch}}, as it dowes not allow interception nor any
+other type of traffic management in an ATSSS UPF.
```
Comment (boucadair) [inline draft-quic-atsss-reqs.mkd]: This is solution oriented, IMO. It depends how I-D.deconinck is used.

**Review by obonaventure — APPROVED** [review]
I'm fine with that start. If we mention UPF, we need to define it before, but that's a detail

**Comment by boucadair** [comment]
> 
> 
> I'm fine with that start. If we mention UPF, we need to define it before, but that's a detail

UPF is already defined in the terminology section.

File: draft-quic-atsss-reqs.mkd [hunk 3]
```diff
@@ -306,19 +312,41 @@ This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no
...
+
+In consequence it is very likely that such traffic (UDP or a subset like QUIC)
+has to be encapsulated in a multipath capable protocol between UE and UPF, which
+meets the ATSSS demands on higher-layer steering. One of the key requirements
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd:353]: Lower-layer steering?

Comment (markusa) [reply draft-quic-atsss-reqs.mkd:353]: No, the multipath protocol has to meet the ATSSS requirements on higher layer steering. This statement mainly focus on the splitting capabilitiy.

File: draft-quic-atsss-reqs.mkd (hunk 1 above)
Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: I don't understand the background of your question.

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: The text says that it "raises issues by its nature of an end-to-end encrypted protocol". My comment is that more elaboration is needed given that "end-to-end encrypted" TLS can benefit from ATSSS:  

Please check figure 3 of https://tools.ietf.org/html/draft-ietf-tcpm-converters-19

File: draft-quic-atsss-reqs.mkd (hunk 2 above)
Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: OK, maybe that needs clarification. What I meant was that an e2e (UE<->Internet Service) MP-QUIC cannot be intercepted and converted like a e2e MPTCP.

File: draft-quic-atsss-reqs.mkd (hunk 1 above)
Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: OK got it. Do you think it's enough to add:

`...end-to-end encrypted protocol including connection metadata.`

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: That is better: s/metadata/control signals.

Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: Solved in https://github.com/obonaventure/draft-quic-atsss-reqs/pull/29/commits/95a6d02e76b980d287dcfdf34b6a1925d771bf27

File: draft-quic-atsss-reqs.mkd (hunk 2 above)
Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: Clarified in https://github.com/obonaventure/draft-quic-atsss-reqs/pull/29/commits/c9814766ec725d0dcca6e46ff7452ccd60a6f1de

File: draft-quic-atsss-reqs.mkd (hunk 3 above)
Comment (markusa) [reply draft-quic-atsss-reqs.mkd:353]: since no further comment arose, I set the state to "resolved". However, the discussion can be resumed at any time.

**Pull request merged**